src/components/Book.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import store from '../reducers';
+
+const state = store.getState();
+const { books } = state.bookReducer;
```

Comment:
- [ ] Kindly do not call the book state here since redux makes the state globally available to all components. Instead, the Book component should accept a `Book` object as a prop [screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/Book/AccceptBookObjectAsProp.png)